editoast/editoast_models/src/tables.rs

```diff
@@ -671,6 +671,8 @@ diesel::table! {
         search_window_begin -> Timestamptz,
         search_window_end -> Timestamptz,
         temporary_speed_limit_group_id -> Nullable<Int8>,
+        enabled_from -> Timestamptz,
+        enabled_to -> Timestamptz,
```

Comment:
We could solve our problem with only an expiration date, I don't know if that was considered in the refinement

Reply:
We didn't think of that. But would the criterion be to take all environments that haven't expired and take the one with the expiration date closest to `now()`?

Reply:
IMO, even if it's okay semantically, It makes the sql more flaky